I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 141, in script
    instruction "Help the spider take the three ladybugs across the lake. The spider can only take one ladybug at a time, but Reddie and Greenie can’t be left alone together and Greenie and Bluey can’t be left alone together. Choose which ladybug the spider should take across the lake."
Exception: Sayer 'instruction' is not defined.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 141, in script
    instruction "Help the spider take the three ladybugs across the lake. The spider can only take one ladybug at a time, but Reddie and Greenie can’t be left alone together and Greenie and Bluey can’t be left alone together. Choose which ladybug the spider should take across the lake."
  File "C:\Users\Linh\Downloads\renpy-8.0.3-sdk\renpy\ast.py", line 881, in execute
    who = eval_who(self.who, self.who_fast)
  File "C:\Users\Linh\Downloads\renpy-8.0.3-sdk\renpy\ast.py", line 774, in eval_who
    raise Exception("Sayer '%s' is not defined." % who)
Exception: Sayer 'instruction' is not defined.

Windows-10-10.0.19045 AMD64
Ren'Py 8.0.3.22090809
deBUG: The Last Temoc 1.0
Sat Feb 11 23:33:36 2023
